lib/resources/r4/encounter.yaml

```diff
@@ -0,0 +1,399 @@
+---
```

Comment:
looks like a few proprietary codeset examples are missing:
identifier.type: codeSet/319
hospitalization.dietPreference: codeSet/18
hospitalization.specialCourtesy: codeSet/67
hospitalization.specialCourtesy: codeSet/16

Reply:
Added an example for identifier.type: https://github.com/cerner/fhir.cerner.com/pull/239/commits/40b3c670db01bc6b48dbb783e357036bee956549

Encounter doesn't write the other hospitalization fields.